modules/create-an-okta-app.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-user-access/configure-okta-identity-cloud.adoc
+:_module-type: PROCEDURE
+[id="create-an-okta-app_{context}"]
+= Creating an Okta app
+
+Before you can use Okta as a SAML 2.0 identity provider for {product-title}, you must create an Okta app.
+
+[WARNING]
+====
+Okta's *Developer Console* does not support the creation of custom SAML 2.0 applications.
+If you are using the Developer Console, you need to first switch to the Admin Console (Classic UI).
+If you see *Developer Console* in the top left of the page, click it and select *Classic UI* to switch.
```

Comment:
Suggest another wording to avoid "if you see". Perhaps:

`To switch, click *Developer Console* in the top left of the page and select *Classic UI*.`